CONTRIBUTING.md

```diff
@@ -161,3 +161,49 @@ Should a new addition be made to the Lottie specifications or a Lottie library f
 Libraries seek to be compliant, which means they might discourage changes that cause them to behave differently from the specifications. However, they also encourage pull requests for changes that accompany an RFC _Proposal_ or _Draft_. Proposals won't be _Accepted_ until it has experience being implemented in a Lottie library.
 
 To allow a library to remain compliant to the specifications while also implementing _Proposals_ and _Drafts_, the library's maintainers may request that these new features are disabled by default with opt-in options, or they may simply wait to merge a well-tested pull request until the Proposal is _Accepted_.
+
+
+## Specification Release Process
+
+The file format specifications are published using a semantic versioning system.
+
+Each published version of the specifications is identified by a unique sequence of major, minor, and patch numbers, following these guidelines:
+
+* Increasing the major version number signals the possibility of breaking changes and significant incompatibilities.
+* Increasing the minor version number happens when new features are introduced, but existing features should not be changed in an incompatible way.
+* The patch version is used to publish minor changes that improve the clarity of the specifications without introducing new functionality.
+
+Each version through a release process consisting of multiple stages to ensure all stakeholders have a say before a new version of the specification is published.
+
+Different versions of the specification might be in different release stages at any given time.
+
+### Active Development
+
+This is the initial stage, new features and proposals are welcome.
+
+Whether or not backwards-incompatible changes are accepted depends on the target version as described above.
+
+
+### Requesting Comments
```

Comment:
Should we specify/discuss minimum time periods for the various stages? This would be to ensure all interested parties have time to review and we can give timelines on when they should review by.

Reply:
Ye I think comments / feature freeze should have a minimum time

Reply:
We should consider different lengths depending on release type (major/minor/patch)

Agreed in LAC for major- 3 weeks comments, 1 week freeze